lec01-introduction.Rmd

```diff
@@ -2,3 +2,171 @@
+
+Furthermore, if we want to change the style of our output document, it is quite easy to do so. Let's add a subheading first, then to keep track of things we want a table of contents, _knit_. Now we realize that we want to include numbers in the report to keep with the departmental guidelines, _knit_. We could alter the style of our report by adding a theme, _knit_. And finally we remember that we need to hand in our assignments as pdf-files, so let's create a pdf instead of a html-document, _remove themee and knit_.
+
+```yaml
+---
+title: "HTML output demo"
+output:
+  html_document:
+    toc: true
+    number_section: true
+    theme: united
+---
+```
+## R Notebook interactivity
+That explains what R markdown is and how we generate reports from it. But how do we work with it interactively? Well it is actually quite simple. Execute the chunks of R code by pressing this little green arrow or the corresponding keyboard shortcut `Shift+Ctrl+Enter`. The output appears right under the code block. This is a great way to perform exploratory data analysis, since you can do your analysis and write comments and conclusions write under it in the same document. To get familiar with this workflow, I will go type in some of the commands that we went through earlier and write comments while working with them. A powerful feature of this workflow, is that there is no extra time needed for code documentation and note-taking, you're doing your analyses and taking notes at the same time, it is great for both taking notes at lectures and to have as a reference when you return to your code in the future.
+
+_Show some examples depending on time left..._
+
+Hopefully, this has given you an idea of how powerful and time-saving R can be for your data analysis, and how we will work with it in this class using the R Notebook. The first assignment will be available on blackboard today, and it is includes a few questions about RStudio and R Markdown. 1 / 4 marks on this assignment is to fill in a survey about your background knowledge. This is so that we can adjust the pace and content of the lectures and to divide you into groups accordingly for the class project later this fall. Please take 10 min to do so now by going to this page (send page to everyone's computer via the teaching software), or if you have to run, you can complete it at home.
+
+
+# Misc notes (not in lecture at the moment)
```

Comment:
I'd suggest leaving this out. You will easily be out of time I think.

Reply:
Remember it is not code along though....

Reply:
Yes, I know. But it still is better to have less and be focused in all material, and instead add well-developed references/resources they can read on their own time if they want.

Reply:
If you don't talk about it during the lecture, it should not be included in the material. The less the students have to worry about the better they will learn the concepts we want them to learn.

Reply:
Yes, I will have the misc part in some additional paper that I will include if there is time. We can let them go early also, just not 40 min early ...